packages/common/src/dto-types.ts

```diff
@@ -76,20 +76,34 @@ export class UpdatePlanDto {
 }
 
 export class CreateStudentDto {
+  @IsNotEmpty()
+  @IsEmail()
+  email: string;
+
   @IsNotEmpty()
   @IsString()
-  fullName: string;
+  password: string;
+}
+
+export class UpdateStudentDto {
```

Comment:
is this used anywhere? i saw that the update endpoint now takes a `Partial<Student>`